receiver/gitproviderreceiver/metadata.yaml

```diff
@@ -11,119 +11,121 @@ status:
   codeowners:
     active: [adrielp, andrzej-stencel]
 
-# this might need to be unique per sub receiver implementation
 resource_attributes:
   organization.name:
     enabled: true
-    description: Git Organization or Project Name
+    description: VCS Organization
     type: string
-  git.vendor.name:
+  vcs.vendor.name:
```

Comment:
Thie resource attribute rename isn't in the changelog, should it be?

`git.vendor.name` -> `vcs.vendor.name`